docs/contracts/events.rst

```diff
@@ -138,7 +138,7 @@ as topics. The event call above can be performed in the same way as
 
 ::
 
-    pragma solidity >=0.4.10 <0.7.0;
+    pragma solidity >=0.4.11 <0.7.0;
```

Comment:
Can you explain the version change? It is a bit weird to have an off-by-one error in this and the next example.

Reply:
I added it when I was dealing with it first time. First available release on github is 0.4.11

Reply:
Since I solved it in script in other way, I will revert it here